spyderlib/plugins/editor.py

```diff
@@ -1521,7 +1521,7 @@ def _clone_file_everywhere(self, finfo):
             editor = editorstack.clone_editor_from(finfo, set_current=False)
             self.register_widget_shortcuts("Editor", editor)
     
-    @Slot()
+#    @Slot(str)

```

Comment:
We're in a bit of a hurry, so I'll do the fix myself in master :-)

Thanks a lot for your contribution, I verified that it does the right thing!